packages/fhirtypes/dist/SmartAppLaunch.d.ts

```diff
@@ -0,0 +1,58 @@
+/*
+ * Generated by @medplum/generator
+ * Do not edit manually.
+ */
+
+import { Encounter } from './Encounter';
+import { Meta } from './Meta';
+import { Patient } from './Patient';
+import { Reference } from './Reference';
+
+/**
+ * User specific configuration for the Medplum application.
+ */
+export interface SmartAppLaunch {
+
+  /**
+   * This is a SmartAppLaunch resource
+   */
+  readonly resourceType: 'SmartAppLaunch';
+
+  /**
+   * The logical id of the resource, as used in the URL for the resource.
+   * Once assigned, this value never changes.
+   */
+  id?: string;
+
+  /**
+   * The metadata about the resource. This is content that is maintained by
+   * the infrastructure. Changes to the content might not always be
+   * associated with version changes to the resource.
+   */
+  meta?: Meta;
+
+  /**
+   * A reference to a set of rules that were followed when the resource was
+   * constructed, and which must be understood when processing the content.
+   * Often, this is a reference to an implementation guide that defines the
+   * special rules along with other profiles etc.
+   */
+  implicitRules?: string;
+
+  /**
+   * The base language in which the resource is written.
+   */
+  language?: string;
+
+  /**
+   * Optional patient indicating that the app was launched in the patient
+   * context.
+   */
+  patient?: Reference<Patient>;
```

Comment:
Will this always be a patient? Or should we consider making this a "ProfileResource" (patient, practioner, relatedperson)